### ISSUE TITLE
[A11y] Work Show Page-All - Print button

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2022]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n
https://staging.oregondigital.org/concern/images/1g05fb620

**Low priority**

The "print" button is a link that functions as a button. It would be better if it were a real button. Accessibility Insights flags this as an error, but my understanding is that it's a best practice, not any WCAG failure.

### Suggested Fix

Change the markup to use a button if it's easy. Simply close this ticket if not.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Verify the "print" button is a button.
